src/main/resources/amenity-data.yml

```diff
@@ -2,4 +2,23 @@ amenity-data:
   amenityConfigs:
     - amenity: scratching post
     - amenity: toys
-    - amenity: garden
\ No newline at end of file
+    - amenity: garden
+    - amenity: litter tray
+    - amenity: specious cage
+    - amenity: aquarium maintenance
```

Comment:
Imo raczej to powinien być standard więc bym tego nie dawał xd

Reply:
chodzi o ` - amenity: aquarium maintenance`, (github 3 linijki zaznacza)

Reply:
xD mozna